pywbem/_cim_types.py

```diff
@@ -817,26 +817,37 @@ def __hash__(self):
 class CIMInt(CIMType, int):
     """
     Base type for CIM integer data types. Derived from :class:`~pywbem.CIMType`
-    and :class:`py3:int` (for Python 3) or :class:`py2:long` (for Python 2).
+    and :class:`py3:int`. The input parameters consistent with :class:`py3:int`
+    except that `x` may be either a positional or keyword input parameter.
```

Comment:
Since we now support Python>=3.8, int()  does not have the x parameter anymore, so we need to ontroduce it instead of referring to it.

How about changing this to:
```
except that the value can also be passed
as a keyword argument `x`.
```

Reply:
Good catch